Makefile

```diff
@@ -343,6 +343,10 @@ clean:
 check-signed-off:
 	$(Q) ./tools/check-signed-off.sh
 
+.PHONY: check-signed-off
+build-zephyr:
```

Comment:
I am not sure what I should put there. I just follosed the example of check-signed-off and did the same with build-zephyr. It basically builds and runs (to test) it.